### PR TITLE
Exercise-14 -mohammed

### PR DESCRIPTION
**Pull request by mashraee**
please review my exercise number 14 and let me know if something was wrong.

### CODE REVIEW

File: Mohammed/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+// 1. NaN
+
+// a-
+// NaN stands for 'Not A Number'
+
+// b-
+let checkForNaN = 'hello' * 1;
+console.log((checkForNaN !== NaN) ?
```
Comment (ciamac-da) [inline Mohammed/index.js]: The result should be true or false and not NaN
like for example: console.log(isNaN("hello" *1));

File: Mohammed/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// a-
+// NaN stands for 'Not A Number'
+
+// b-
+let checkForNaN = 'hello' * 1;
+console.log((checkForNaN !== NaN) ? 
+`The expression ("hello" * 1) is a ${checkForNaN}` : `The expression ("hello" * 1) is a ${checkForNaN}`);
+
+// c-
+checkForNaN = '100em' *1;
```
Comment (ciamac-da) [inline Mohammed/index.js]: Here also the same. We wanna check if the result is true or false! You also dont need *1 in this task!
console.log(isNaN("100em"));

File: Mohammed/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+console.log(numberWithRoundedFraction);
+
+// b-
+numberWithFloatingFraction = 0.63783990;
+numberWithRoundedFraction = numberWithFloatingFraction.toFixed(1);
+console.log(numberWithRoundedFraction);
+
+// c-
+numberWithFloatingFraction = 1.842220;
+numberWithRoundedFraction = numberWithFloatingFraction.toFixed(6);
```
Comment (ciamac-da) [inline Mohammed/index.js]: toFIxed(4) and not 6!

**Review by ciamac-da — CHANGES_REQUESTED** [review]

**Review by ciamac-da — APPROVED** [review]
🥇

File: Mohammed/index.js (hunk 2 above)
Comment (mashraee) [reply Mohammed/index.js]: Hi Ciamac, thank you for your feedback. I forgot about the isNaN() method so I went to reinvent it!
I pushed the refined code to github.

**Review by ciamac-da — APPROVED** [review]
Thanks mohammed🥇

**Pull request merged**